osmaxx-py/osmaxx/excerptexport/static/excerptexport/libraries/Leaflet.Spin/leaflet.spin.js

```diff
@@ -0,0 +1,41 @@
+L.SpinMapMixin = {
+    spin: function (state, options) {
```

Comment:
Avoid flag arguments (smell/heuristic **F4**, in the 'Clean Code' book) / selector arguments (smell/heuristic **G15**). As you never call this with a state that isn't a boolean literal, split this function into two: `start_spinning` and `stop_spinning`.